### ISSUE TITLE
BTC transfer doesn't work

### ISSUE THREAD

**Issue by rexslatz** [LiskHQ/lisk-desktop#2659]
### Description
i am using mac os. trying to send bitcoin from bitcoin wallet. on the transaction summary page when i click on the send button (bitcoin wallet) nothing happens. 

### Expected behavior
I would expect transaction confirmed screen

### Actual behavior  
nothing happend when send bitcoin button clicked

### Steps to reproduce
Follow the send bitcoin process in lisk desktop

### Which version(s) does this affect? (Environment, OS, etc...)
Both maOS and Windows

**Comment by Nazgolze** [comment]
I spoke with @rexslatz on Discord and we have a few additional details.
```please help. i cannot send bitcoin - neither with mobile app (iphone) nor desktop app. i've raised an issue re. desktop app - problem is both with Windows and macOS. Re. mobile app - if I try to transfer entire balance i get an error message 'your balance is not sufficient'. If i reduce the amount to transfer i get error message 'undefined is not an object (evaluating 'k.value')'```

I'm able to reproduce part of this issue in lisk-desktop (1.22) in testnet. I tried sending my entire balance and it did not take into account the transaction fee, so the submit button indeed did nothing in that case. When I manually reduced the amount I wanted to send to account for it, it worked for me. I'm running the linux version though, so that could possibly account for the difference. My calculation was also quite accurate, perhaps if I reduced the sending amount by an insufficient margin, it might be possible to reproduce the 2nd part of the issue
